zcash_client_sqlite/src/wallet.rs

```diff
@@ -129,6 +131,32 @@ pub(crate) mod scanning;
 
 pub(crate) const BLOCK_SAPLING_FRONTIER_ABSENT: &[u8] = &[0x0];
 
+enum AccountType {
```

Comment:
Document this type; in particular that it is a database type marker and should not be made public.

(We have a few other enums that are public and became database type markers; this imposes stability requirements that are hard to track, but fortunately for most of those existing types the markers have some externally-specified typecode to use. That is not the case here, so we should just avoid the numeric mapping becoming public.)